docker/test.sh

```diff
@@ -0,0 +1 @@
+xvfb-run -a npm test
```

Comment:
probably belongs in `script` rather than `docker` directory, but up to you

Reply:
I'm happy with it where it is. It's a single line script which is only ever called by the docker-compose file